src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -249,16 +275,22 @@ public async Task ProcessInputAsync(Message message, Flow flow, CancellationToke
                                 }
                             }
                         } while (!stateWaitForInput);
-                        
+
                         // Process the global output actions
                         if (flow.OutputActions != null)
                         {
                             await ProcessActionsAsync(lazyInput, context, flow.OutputActions, inputTrace?.OutputActions, linkedCts.Token);
                         }
+
+                        // Schedule expiration time if input is configured
+                        if (state.InputExpirationTimeEnabled())
+                        {
+                            await _schedulerExtension.ScheduleMessageAsync(message.CreateInputExirationTimeMessage(), DateTimeOffset.UtcNow.AddMinutes(state.Input.WaitInputExpirationTimeMinutes.Value), linkedCts.Token);
```

Comment:
All extensions send commands on behalf of router when the **use router context** flag is active. Is important to test this scenario.

Reply:
I create a decorator of ISchedulerExtension that removes OwnerContext send schedule commands and recreate OwnerContext.